include/grpc++/support/auth_context.h

```diff
@@ -72,20 +72,26 @@ class AuthPropertyIterator
   const char* name_;
 };
 
+/// Class encapsulating the Authentication Information.
+/// 
+/// It includes the secure identity of the peer, the type of secure transport
+/// used as well as any other properties required by the authorization layer.
```

Comment:
I could add those as part of this PR or as a separate one (to keep this one purely documentational).